source/Symbol/SwiftASTContext.cpp

```diff
@@ -4556,8 +4557,8 @@ swift::SILModule *SwiftASTContext::GetSILModule() {
   VALID_OR_RETURN(nullptr);
 
   if (m_sil_module_ap.get() == NULL)
-    m_sil_module_ap = swift::SILModule::createEmptyModule(GetScratchModule(),
-                                                          GetSILOptions());
+    m_sil_module_ap = swift::SILModule::createEmptyModule(
+                                                          GetScratchModule(), GetSILOptions(), swift::PrimarySpecificPaths("", "<lldb>"));
```

Comment:
Can you please `clang-format` your changes?

Reply:
Yup, sorry for the oversight.